app/views/dit_landing_page/_page_header.html.erb

```diff
@@ -15,19 +15,26 @@
 
 <header class="dit-landing-page__header--with-bg-image">
   <div class="dit-landing-page__header-text-block">
-    <div class="govuk-width-container landing-page_header-text-container">
+    <div class="govuk-width-container">
       <div class="govuk-grid-row">
-        <div class="govuk-grid-column-two-thirds">
+        <div class="govuk-grid-column-two-thirds dit-landing-page_header-text-container <%= 'dit-landing-page_header-text-container--no-translation-nav' unless defined?(presented_taxon) %>">
```

Comment:
Could you help me understand the styling variation for no translation nav, and why it depends on a presented taxon being defined? The two sounds quite unrelated to me - maybe you could add something in the commit message? (Perhaps we could check `presented_taxon.translation_links` instead.)

Reply:
To prevent the page layout from moving around, when we don't have the translation navigation we add extra padding to the header container. Indeed `presented_taxon.translation_links` might be a better check. I'm rather sceptical to introducing the translation navigation with this work, as it doesn't seem related. So I'll amend the commit to remove it from this PR. The modifier class `dit-landing-page_header-text-container--no-translation-nav` would still be valuable until we get the navigation in.